std/format.d

```diff
@@ -3906,6 +3906,43 @@ if (isPointer!T && !is(T == enum) && !hasToString!(T, Char))
     }
 }
 
+/**
+   SIMD vectors are formatted as arrays.
+ */
+void formatValue(Writer, V, Char)(auto ref Writer w, V val, const ref FormatSpec!Char f)
+if (isSIMDVector!V)
+{
+    formatValue(w, val.array, f);
+}
+
+unittest
+{
+    version (OSX)
+    {
+        version (X86)
+        {
+            import core.simd;
+            static if (is(float4))
+            {
+                //float4 f;
```

Comment:
After running remote code for 3 hours it appears that there's a codegen error or something, see https://issues.dlang.org/show_bug.cgi?id=17823.

When the declaration of `f` is commented the test for OSX 32 bits doesn't crash anymore.

Reply:
Even with this commented out it's segfaulting on CircleCI.

Reply:
@ZombineDev: can you mark it as blocked please ? Also remove the polution from CodeCov